README.md

```diff
@@ -2,6 +2,17 @@
 
 ## 📚 Docs 👉🏻 <https://nextflow-io.github.io/nf-validation>
 
+<!-- prettier-ignore -->
+> [!IMPORTANT]
+> **`nf-validation` has now been renamed to `nf-schema`. The `nf-validation` plugin will not receive any future updates.**
+> **Please update your pipelines to use [`nf-schema`](https://github.com/nextflow-io/nf-schema) instead.**
+>
+> This change was necessary to prevent older versions of `nf-core` pipelines from with unpinned plugin references from breaking when updating to the latest version of `nf-validation`.
+> **Please pin the version of `nf-schema` in your pipeline's `nextflow.config` file:**
+> ```nextflow
+> plugins { id 'nf-schema@2.0.0' }
+> ```
+
 **A Nextflow plugin to work with validation of pipeline parameters and sample sheets.**
```

Comment:
```suggestion
<!-- prettier-ignore -->
> [!IMPORTANT]
> **`nf-validation` has now been renamed to `nf-schema`. The `nf-validation` plugin will not receive any future updates.**
> **Please update your pipelines to use [`nextflow-io/nf-schema`](https://github.com/nextflow-io/nf-schema) instead.**
>
> This change was necessary to prevent older versions of `nf-core` pipelines from with unpinned plugin references from breaking when updating to the latest version of `nf-validation`.
> **Please pin the version of `nf-schema` in your pipeline's `nextflow.config` file:**
> ```nextflow
> plugins { id 'nf-schema@2.0.0' }
> ```

## 📚 Docs 👉🏻 <https://nextflow-io.github.io/nf-validation>

**A Nextflow plugin to work with validation of pipeline parameters and sample sheets.**
```

Reply:
Yeah indeed, I've not found a better solution yet :/